cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -87,6 +87,22 @@ bool gdv_fn_in_expr_lookup_int64(int64_t ptr, int64_t value, bool in_validity) {
   return holder->HasValue(value);
 }
 
+bool gdv_fn_in_expr_lookup_float(int64_t ptr, float_t value, bool in_validity) {
```

Comment:
Can you replace float_t with float. Thats what likely causing the issue.
Same for doube_t.

Reply:
Fixed, sorry I thought I already had changed every one.